sql/updates/world/master/9999_99_99_99_world.sql

```diff
@@ -0,0 +1,575 @@
+(12090,29443,0,'',52649),
+(12091,29446,0,'',52649),
+(12092,29447,0,'',52649),
+(12164,29660,0,'',52649),
+(15615,39167,0,'',52649);
+
+DELETE FROM `conversation_actors` WHERE `ConversationId` IN (12090,12091,12092,12164,15615);
+INSERT INTO `conversation_actors` (`ConversationId`,`ConversationActorId`,`ConversationActorGuid`,`Idx`,`CreatureId`,`CreatureDisplayInfoId`,`NoActorObject`,`ActivePlayerObject`,`VerifiedBuild`) VALUES
+(12090,0,0,0,0,0,0,1,52649), -- Player
+(12090,71372,0,1,0,0,0,0,52649), -- Captain Garrick
+(12091,71587,8000023,0,0,0,0,0,52649), -- Henry Garrick
+(12092,0,0,0,0,0,0,1,52649), -- Player
+(12092,71372,0,1,0,0,0,0,52649), -- Captain Garrick
+(12164,71372,0,0,0,0,0,0,52649), -- Captain Garrick
+(15615,71372,0,0,0,0,0,0,52649), -- Captain Garrick
+(15615,78493,0,1,0,0,0,0,52649); -- Henry Garrick
+
+DELETE FROM `conversation_line_template` WHERE `Id` IN (29443,29444,29446,29660,29447,29448,29450,39167,39163,39164,39165,39166);
+INSERT INTO `conversation_line_template` (`Id`,`UiCameraID`,`ActorIdx`,`Flags`,`VerifiedBuild`) VALUES
+(29443,0,0,0,52649),
+(29444,0,1,0,52649),
+(29446,0,0,0,52649),
+(29660,0,0,0,52649),
+(29447,0,0,0,52649),
+(29448,0,1,0,52649),
+(29450,0,1,0,52649),
+(39167,0,0,0,52649),
+(39163,0,1,0,52649),
+(39164,0,0,0,52649),
+(39165,0,0,0,52649),
+(39166,0,1,0,52649);
+
+-- Lindie Springstock SAI 'Ogre Ruins'
+SET @NPC := 150245;
+UPDATE `creature_template` SET `AIName`='SmartAI' WHERE `entry`=@NPC;
+DELETE FROM `smart_scripts` WHERE `entryorguid` IN (@NPC) AND `source_type`=0;
+INSERT INTO `smart_scripts` (`entryorguid`,`source_type`,`id`,`link`,`event_type`,`event_phase_mask`,`event_chance`,`event_flags`,`event_param1`,`event_param2`,`event_param3`,`event_param4`,`event_param5`,`action_type`,`action_param1`,`action_param2`,`action_param3`,`action_param4`,`action_param5`,`action_param6`,`target_type`,`target_param1`,`target_param2`,`target_param3`,`target_param4`,`target_x`,`target_y`,`target_z`,`target_o`,`comment`) VALUES
+(@NPC,0,0,1,1,0,100,0,8000,10000,8000,10000,0,66,0,0,0,0,0,0,19,156808,10,0,0,0,0,0,0,'OOC Face Small Boar'),
+(@NPC,0,1,0,61,0,100,0,0,0,0,0,0,5,4,0,0,0,0,0,1,0,0,0,0,0,0,0,0,'OOC Emote OneShotCheer');
+
+-- Captain Garrick SAI 'Ogre Ruins'
+SET @NPC := 156807;
+UPDATE `creature_template` SET `AIName`='SmartAI' WHERE `entry`=@NPC;
+DELETE FROM `smart_scripts` WHERE `entryorguid` IN (@NPC) AND `source_type`=0;
+INSERT INTO `smart_scripts` (`entryorguid`,`source_type`,`id`,`link`,`event_type`,`event_phase_mask`,`event_chance`,`event_flags`,`event_param1`,`event_param2`,`event_param3`,`event_param4`,`event_param5`,`action_type`,`action_param1`,`action_param2`,`action_param3`,`action_param4`,`action_param5`,`action_param6`,`target_type`,`target_param1`,`target_param2`,`target_param3`,`target_param4`,`target_x`,`target_y`,`target_z`,`target_o`,`comment`) VALUES
+(@NPC,0,0,47,0,0,100,0,55194,0,0,0,0,11,296845,0,0,0,0,0,7,0,0,0,0,0,0,0,0,'Quest reward Cast Gear repair on player');
```

Comment:
you mixed `link` and `event_type` here.

Can you please please please start using the tools we are using for SAI? It prevents such goofy issues caused by writing by hand and you directly get the comment without extra work - you dont have to type it in by hand. Also you dont have to look into wiki all day to write any lines, its literally win-win.
The tool im talking about is https://github.com/bandysc/wowdatabaseeditor